app/pages/faq.cjsx

```diff
@@ -0,0 +1,24 @@
+counterpart = require 'counterpart'
+React = require 'react'
+{Markdown} = (require 'markdownz').default
+
+counterpart.registerTranslations 'en',
+  faqPage:
+    content: '''
+
+      ### Frequently Asked Questions
+
+      - **What browser version does Zooniverse support?**
+        We support major browsers up to the second to last version.
```

Comment:
I didn't include that because I'm not sure if Edge and IE are considered separate browsers...